main.go

```diff
@@ -3,10 +3,10 @@ package main
 import (
 	"fyne.io/fyne/v2"
 	"fyne.io/fyne/v2/app"
+	"github.com/Jacalz/rymdport/v3/internal/updater"
 
 	"github.com/Jacalz/rymdport/v3/internal/assets"
 	"github.com/Jacalz/rymdport/v3/internal/ui"
-	"github.com/Jacalz/rymdport/v3/internal/updater"
```

Comment:
Please move the line back to where it were. I prefer to have the imports formatted in that way.